swedish-oidc-claims-specification.md

```diff
@@ -390,104 +389,155 @@ There are basically two ways for a Relying Party to indicate which claims it wis
+defines the `https://id.oidc.se/scope/naturalPersonInfo` scope to limit the amount of user identity
+claims to what is offered using a Swedish eID.
 
 <a name="natural-person-identity-personal-number"></a>
 ### 3.2. Natural Person Identity - Personal Number
 
 **Scope:** `https://id.oidc.se/scope/naturalPersonNumber`
 
-**Description:** The scope extends the `https://id.oidc.se/scope/naturalPersonName` scope with a Swedish civic
-registration number (personnummer) or a Swedish coordination number (samordningsnummer).
-
-| Claim | Description/comment | Reference | Requirement |
-| :--- | :--- | :--- | :--- |
-| `https://id.oidc.se/claim/`<br />`personalIdentityNumber` | Swedish civic registration number. | This profile | Mandatory<sup>1</sup> |
-| `https://id.oidc.se/claim/`<br />`coordinationNumber` | Swedish coordination number. If delivered according to this scope, the coordination number SHOULD have a status of Active<sup>2</sup>. | This profile | Mandatory<sup>1</sup> |
-| `family_name` | Surname/family name | \[[OpenID.Core](#openid-core)\] | Mandatory |
-| `given_name` | Given name | \[[OpenID.Core](#openid-core)\] | Mandatory |
-| `name` | Display name | \[[OpenID.Core](#openid-core)\] | Mandatory | 
-| `birthdate` | Date of birth | \[[OpenID.Core](#openid-core)\] | Optional | 
-
-> **\[1\]**: A `personalIdentityNumber` OR `coordinationNumber` claim MUST be delivered, but not both.
-
-> **\[2\]**: A Swedish coordination number has a status associated, where the different statuses are "active" (aktivt),
-> "on hold" (vilandeförklarat), "on hold - closed" (vilandeförklarat - stängt) and "deregistered" (avregistrerat). The reason that
-> this profile does not apply a "MUST"-requirement for an active status is that it is a too strict requirement to put on claims
-> issuers. In theory they would have to make controls against the Swedish population register every time an assertion or token would
-> be released. Therefore, a consumer of the `coordinationNumber` scope MUST always ensure that the number's status is acceptable.
+**Description:** A scope that requests a Swedish civic registration number (personnummer) or 
+a Swedish coordination number (samordningsnummer). These identity numbers are often used as 
+primary identities at public Swedish organizations.
+
+Note that an OpenID Provider delivering claims according to this scope MUST NOT deliver both a
+`personalIdentityNumber` and a `coordinationNumber` claim. These are mutually exclusive. However, 
+a person that holds a civic registration number (personnummer) may previously have had a 
+coordination number. This number may then be delivered using the `previousCoordinationNumber` 
+claim (see section [2.1.2.2](#previous-coordination-number)). This claim has to be explicitly
+requested using the `claims` request parameter and is not part of the `naturalPersonNumber` scope.
+
```

Comment:
This place was not fixed. Change to:

"This claim **can** be explicitly requested using the `claims` request parameter"